TurkishFilter/sections/specific.txt

```diff
@@ -3006,7 +3007,7 @@ medyaradar.com###side iframe[src^=" http://www.emlakeki.com/"][src*="banner.aspx
 filmmakinesi.pw,filmakinesi.com,filmakinesi.net###single_1_aciklama_alt
 filmmakinesi.pw,filmakinesi.com,filmakinesi.net###film > div[class*="-metin"]
 bafrahaber.com##.menuustu-reklam
-bafrahaber.com###sag-reklam-up
+bafrahaber.com##.manset_alti_reklam_mk
```

Comment:
I don't see this element on my end, so I removed it. If you have it, please send a screenshot, I'll add this directly.